lib/prefab/client.rb

```diff
@@ -97,6 +92,20 @@ def reset!
       @_channel = nil
     end
 
+    def enabled?(feature_name, lookup_key=nil, attributes={})
+      feature_flag_client.feature_is_on_for?(feature_name, lookup_key, attributes: attributes)
+    end
+
+    def get(key, default_or_lookup_key=NO_DEFAULT_PROVIDED, attributes={}, ff_default=NO_DEFAULT_PROVIDED)
+      result = config_client.get(key, default_or_lookup_key)
+
+      if result.is_a?(Prefab::FeatureFlag)
+        feature_flag_client.get(key, default_or_lookup_key, attributes, default: ff_default)
+      else
+        result
+      end
+    end
```

Comment:
This is now pulling double-duty by returning either a config value or the value of a flag. Is this too clever?

I don't think so but I'm happy to hear different opinions.

I think, where possible, we should consistently return a value with `get`. Without this cleverness, we could return a `Prefab::FeatureFlag` instance if you called `$prefab.get(key)` with a key that is a flag's key.

Reply:
🧇   I can see both sides. 

Should there just be another method beside this `$prefab.feature(feature, lookup_key...` ?

We can always add the generalized `get`  later, but hard to take it away.

Reply:
> We can always add the generalized get later, but hard to take it away.

Agreed. This is the part I'm 🧇 on too.

> Should there just be another method beside this $prefab.feature(feature, lookup_key... ?

That's fine, but I still get bad feelings about having a `get` that can return an instance of `Prefab::FeatureFlag` when the user might not expect that. It isn't _wrong_, but it feels like having two different methods adds to the cognitive overhead of using this lib.

🤔 

>  hard to take it away

Definitely hard, but not impossible. We could throw a deprecation warning when you read a FF value through this method and then phase it out eventually.

With a bias towards action, I'm going to merge this for now. We can still take it away before publishing a new gem version. LMK if you have further thoughts!